src/pybind/mgr/cephadm/tests/test_scheduling.py

```diff
@@ -161,17 +165,23 @@ def run_scheduler_test(results, mk_spec, get_hosts_func, get_daemons_func, key_e
 #       |   |   | |     |
 test_explicit_scheduler_results = [
     (k("*   *   0 *"), error(ServiceSpecValidationError, 'num/count must be > 1')),
-    (k("*   e   N l"), error(OrchestratorValidationError, 'Cannot place <ServiceSpec for service_name=mon>: No matching hosts for label mylabel')),
-    (k("*   e   N p"), error(OrchestratorValidationError, 'Cannot place <ServiceSpec for service_name=mon>: No matching hosts')),
-    (k("*   e   N h"), error(OrchestratorValidationError, 'placement spec is empty: no hosts, no label, no pattern, no count')),
+    (k("*   e   N l"), error(OrchestratorValidationError,
+                             'Cannot place <ServiceSpec for service_name=mon>: No matching hosts for label mylabel')),
+    (k("*   e   N p"), error(OrchestratorValidationError,
+                             'Cannot place <ServiceSpec for service_name=mon>: No matching hosts')),
+    (k("*   e   N h"), error(OrchestratorValidationError,
+                             'placement spec is empty: no hosts, no label, no pattern, no count')),
     (k("*   e   * *"), none),
-    (k("1   12  * h"), error(OrchestratorValidationError, "Cannot place <ServiceSpec for service_name=mon> on 2: Unknown hosts")),
-    (k("1   123 * h"), error(OrchestratorValidationError, "Cannot place <ServiceSpec for service_name=mon> on 2, 3: Unknown hosts")),
+    (k("1   12  * h"), error(OrchestratorValidationError,
+                             "Cannot place <ServiceSpec for service_name=mon> on 2: Unknown hosts")),
+    (k("1   123 * h"), error(OrchestratorValidationError,
+                             "Cannot place <ServiceSpec for service_name=mon> on 2, 3: Unknown hosts")),
     (k("1   *   * *"), exactly('1')),
     (k("12  1   * *"), exactly('1')),
     (k("12  12  1 *"), one_of('1', '2')),
     (k("12  12  * *"), exactly('1', '2')),
-    (k("12  123 * h"), error(OrchestratorValidationError, "Cannot place <ServiceSpec for service_name=mon> on 3: Unknown hosts")),
+    (k("12  123 * h"), error(OrchestratorValidationError,
+                             "Cannot place <ServiceSpec for service_name=mon> on 3: Unknown hosts")),
```

Comment:
the scheduler test is already **really** hard to read. breaking the table here makes it even harder to grasp. 

Can we leave this as it was?